geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -251,11 +251,27 @@ pub struct ReplicaBlockInfoV3<'a> {
     pub entry_count: u64,
 }
 
+/// Extending ReplicaBlockInfo by sending the entries_count.
```

Comment:
Nit: can you please update this copy-pasta?

Reply:
Updated!